lib/themes/dosomething/paraneue_dosomething/templates/fact/node--fact_page.tpl.php

```diff
@@ -100,4 +100,8 @@
       </div>
     </div>
   <?php endif; ?>
+
+  <?php if ($info_bar): ?>
```

Comment:
@angaither it's the little bar above the footer that says "In partnership with..."